projects/contributor-experience.md

```diff
@@ -0,0 +1,111 @@
+contribute to this project in different ways.
+
+While in the beginning of the project contributors knew each other well and
+could share implicit in-group knowledge effortlessly, the project has outgrown that
+phase a while ago and we see different kinds of problems, that make it hard
+for long-term contributors to maintain the project sustainably and for new-comers
+to enter the project effortlessly.
+
+In this project proposal we suggest to form a cross-cutting special interest group
+that takes responsibility for driving contributor experience projects and that
+closely collaborates with all other SIGs and governing bodies to increase contributor
+sustainability, velocity and happiness.
+
+### Current challenges
+
+Among others the challenges the OpenTelemetry project is facing in regards of
+contributor experience are:
+
+* No consistent way of moving contributors "up the ladder", e.g. from member to
+  approver, from approver to maintainer etc, which makes those changes invisible
+  to the larger project and opaque to project leadership on who is doing
+  what.
+* No clear onboarding path for new contributors, or contributors taking on new
+  responsibilities within the project (triagers, approvers, maintainers, committee
+  members), which leads to friction in understanding of the overall project goals,
+  to misalignment in implementations and the feeling of contributors being remote
+  to other parts of the project.
+* No explicit contributor guidebook that helps contributors to answer common
+  questions no matter where they are on the contributors' ladder.
+* Mentorship for new contributors or contributors stepping up into new roles to
+  make them part of the project and aware of.  
+* Only implicit and unstructured recognition for contributors, which would help
+  contributors to feel seen and recognized.
+* No structure collaboration across SIGs, especially in the case of cross-cutting
+  concerns (Security, Docs, End-User, ...)
+* SIGs are short of contributors and require some help to advertise their need
+  (blog post, social posts, etc) and to lower the barrier of entry into their
+  sub-project.
+* _add more_
+
+Note that there are individuals within the project who try to tackle some of those
+aspects as individuals and not in a team effort, notable:
+
+* [Add skeleton for maintainer and contributor guidebooks](https://github.com/open-telemetry/community/pull/2051)
+* Mentorship efforts, e.g. [Outreachy](https://cloud-native.slack.com/archives/C060GFUL0P6)
+* Surveys by the End User SIG that aim towards contributor experience
+* _add more_
+
+Also related to this initiative is the [Check-In process of the Governance Committee](https://github.com/open-telemetry/community/blob/main/gc-check-ins.md)
+
+### Goals, objectives, and requirements
+
+The goal of this project is to establish a cross-cutting "SIG Contributor Experience",
+which is "responsible for improving the experience of those who upstream contribute to
```

Comment:
Those who upstream contribute?

Reply:
This is a verbatim copy from the k8s contributor experience charter, I can reword it to "those who contribute upstream"

Reply:
Alright, it sounded weird but it's still comprehensible

Reply:
Since it is a quote I will leave it as is